### ISSUE TITLE
conversion script 'ddl2cpp' does not support negative integer values

### ISSUE THREAD

**Issue by mitsutaka-takeda** [rbock/sqlpp11#54]
When SQL include a negative integer value, ddl2cpp failed to convert the SQL.
```sql
CREATE TABLE TEST (
  id varchar(255) PRIMARY KEY,
  num integer DEFAULT -1
) ENGINE=InnoDB DEFAULT CHARSET=utf8
```

I think we need to add something like this to support negative numbers

```python
negativeSign = Literal('-')
ddlNum     = Combine(Optional(negativeSign) + Word(nums + "."))
```

**Comment by rbock** [comment]
Sounds good!

If you stumble over more things like this, could you make the changes in a fork and submit a pull request?

**Comment by mitsutaka-takeda** [comment]
Sure! I'will.

**Comment by rbock** [comment]
Fixed that in develop.

**Comment by rbock** [comment]
Fixed in release 0.35
